net/fail2ban/LICENSE.txt

```diff
@@ -0,0 +1,369 @@
+The following copyright applies to all files present in the Fail2ban package,
```

Comment:
Where did you get this? You changed name of this license.
It should be COPYING and we dont need to have in this repository the same file, which is included in [fail2ban repository](https://github.com/fail2ban/fail2ban/blob/master/COPYING).

Remove it and change it in the Makefile in PKG_LICENSE_FILES and put there COPYING.